src/to-mils.js

```diff
@@ -1,14 +1,23 @@
 module.exports = (input) => {
-    let numbers = parseFloat(/[\d\.]+/gm.exec(input)[0])
-    let letters = (/[A-z ]+/gm.exec(input)[0]).replace(/[ ]/gm, '')
+    let words = input.split(" and ");
+    for (i=0; i<words.length; i++) {
+        let numbers = parseFloat(/[\d\.]+/gm.exec(words[i])[0])
+        let letters = (/[A-z ]+/gm.exec(words[i])[0]).replace(/[ ]/gm, '')
 
-    if (letters == "day" || letters == "days"){
-        return numbers * 86400000
-    } else if (letters == "hr" || letters == "hrs" || letters == "hour" || letters == "hours"){
-        return numbers * 3600000
-    } else if (letters == "min" || letters == "minute" || letters == "minutes") {
-        return numbers * 60000
-    } else if (letters == "sec" || letters == "second" || letters == "seconds"){
-        return numbers * 1000
+        if (letters == "day" || letters == "days") {
+            words[i] = numbers * 86400000
```

Comment:
replacing the words index with a number feels weird to me, can you instead keep track of a sum and just add to it? This also makes it so we don't have to loop the words twice

Something like 
```js
let sum = 0;
for(...){
    if(...){
        sum += whatever
    }
}
```